public/setup/atomicupdate/update_path.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+$sql[]="UPDATE `{$dbprefix}acces` SET `page`='/statistics' WHERE `page` = 'statistiques/index.php';";
```

Comment:
You can just remove the page (not the entry) and provide the permissions in config/permissions.yaml